backend/tests/test_schema.py

```
@@ -0,0 +1,55 @@
+
+    for expected in expected_calls:
+        assert expected in actual_calls
+
+def test_init_schema_handles_existing_indexes():
+    # Arrange
+    mock_client = MagicMock()
+    mock_graph = MagicMock()
+    mock_client.select_graph.return_value = mock_graph
+
+    # Simulate the first index already existing
+    def side_effect(*args, **kwargs):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `side_effect`

(ANN202)